CHANGELOG.md

```diff
@@ -25,9 +25,12 @@
     - current_database
     - current_session
     - date_trunc
-    - udf
     - object_construct
     - object_construct_keep_null
+    - pow
+    - udf
+    - sproc
```

Comment:
nit: I think udf and sproc are worth being mentioned separately in the changelog